common/persistence/wrappers/templates/metered.tmpl

```diff
@@ -0,0 +1,65 @@
+import (
```

Comment:
Please include a header specifying they're generated else we'll definately have folks editing the generated code (including me probably)

Reply:
Gowrap adds a header:
```
// Code generated by gowrap. DO NOT EDIT.
// template: ../templates/metered.tmpl
// gowrap: http://github.com/hexdigest/gowrap
```